### PR TITLE
modify package.json information field

### PR DESCRIPTION
**Pull request by xmzheng**
Update the package.json to be ready for pushing to npm.

### CODE REVIEW

File: packages/vm/package.json [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-    "url": "https://github.com/ethereumjs/ethereumjs-monorepo/issues?q=is%3Aissue+label%3A%22package%3A+vm%22"
+    "url": "https://github.com/oasislabs/NomicFoundation-ethereumjs-vm/issues"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ethereumjs/ethereumjs-monorepo.git"
+    "url": "https://github.com/oasislabs/NomicFoundation-ethereumjs-vm.git"
   },
   "license": "MPL-2.0",
   "author": "mjbecze <mjbecze@gmail.com>",
```
Comment (xmzheng) [inline packages/vm/package.json:20]: should we change the author fields or just remove them?

**Comment by xmzheng** [comment]
@nhynes

File: packages/vm/package.json (hunk 1 above)
Comment (nhynes) [reply packages/vm/package.json:20]: You can leave the author field in, as it's polite to cite the author.

**Review by nhynes — APPROVED** [review]
You can push this to NPM when you're ready.

**Pull request merged**